arm-datalake-analytics/account/2016-11-01/swagger/account.json

```diff
@@ -1386,9 +1386,6 @@
       "description": "The properties to update that are associated with an underlying Data Lake Analytics account to."
     },
     "AddDataLakeStoreParameters": {
-      "required": [
```

Comment:
It is not required. The properties and parameters are not required and can be null, since the property is just an optional suffix. The generated code was weird, in that it required the user to create an empty object to satisfy the condition, even though the service does nothing with it.